### ISSUE TITLE
App shortcuts broken

### ISSUE THREAD

**Issue by stefan2904** [orgzly-revived/orgzly-android-revived#177]
In the latest (?) release, [app shortcuts](https://developer.android.com/develop/ui/views/launch/shortcuts) stopped working for me. I mean the ones for quick-capture and sync:

![image](https://github.com/orgzly-revived/orgzly-android-revived/assets/527767/7d58f36a-b7a5-447d-9222-262297a4099f)

When clicking on them, I get "App not installed":

![image](https://github.com/orgzly-revived/orgzly-android-revived/assets/527767/04316954-45ff-40fd-9842-bfbc7e5785be)

    Device: Pixel 7
    Android Version: 14 (February 2024)
    Orgzly Revived Version: 1.8.16 (fdroid)

I assume this problem exists in 1.8.15 already.

This might be related to #176, although widgets work for me.

**Comment by stefan2904** [comment]
## logcat

```
I ActivityTaskManager: START u0 {act=com.orgzly.intent.action.SYNC_START flg=0x1000c000 cmp=@2131886121/com.orgzly.android.ui.SyncShortcutActivity bnds=[520,1423][1001,1539]} with LAUNCH_MULTIPLE from uid 10392 (realCallingUid=10201) result code=-92
E LauncherAppsService: Couldn't start activity, code=-92
E NotificationService: Package has already queued 5 toasts. Not showing more. Package=com.google.android.apps.nexuslauncher
E ActivityContext: Unable to launch. tag=WorkspaceItemInfo(id=-1 type=DEEPSHORTCUT container=# com.android.launcher3.logger.LauncherAtom$ContainerInfo@1a1bf6a targetComponent=ComponentInfo{com.orgzlyrevived/com.orgzly.android.ui.LauncherActivity} screen=-1 cell(-1,-1) span(1,1) minSpan(1,1) rank=1 user=UserHandle{0} title=Synchronisieren) intent=Intent { act=android.intent.action.MAIN cat=[com.android.launcher3.DEEP_SHORTCUT] flg=0x10200000 pkg=com.orgzlyrevived cmp=com.orgzlyrevived/com.orgzly.android.ui.LauncherActivity bnds=[520,1423][1001,1539] (has extras) }
E ActivityContext: android.content.ActivityNotFoundException: Shortcut could not be started
E ActivityContext: 	at android.content.pm.LauncherApps.startShortcut(LauncherApps.java:1651)
E ActivityContext: 	at android.content.pm.LauncherApps.startShortcut(LauncherApps.java:1616)
E ActivityContext: 	at com.android.launcher3.views.ActivityContext.startActivitySafely(Unknown Source:125)
E ActivityContext: 	at com.android.launcher3.Launcher.startActivitySafely(Unknown Source:36)
E ActivityContext: 	at com.android.launcher3.uioverrides.QuickstepLauncher.startActivitySafely(Unknown Source:13)
E ActivityContext: 	at com.android.launcher3.touch.ItemClickHandler.startAppShortcutOrInfoActivity(Unknown Source:127)
E ActivityContext: 	at com.android.launcher3.touch.ItemClickHandler.a(Unknown Source:108)
E ActivityContext: 	at com.android.launcher3.touch.e.onClick(Unknown Source:0)
E ActivityContext: 	at com.android.launcher3.uioverrides.QuickstepLauncher.onItemClicked(Unknown Source:12)
E ActivityContext: 	at b1.i.onClick(Unknown Source:2)
E ActivityContext: 	at com.android.launcher3.popup.f.onClick(Unknown Source:12)
E ActivityContext: 	at android.view.View.performClick(View.java:7658)
E ActivityContext: 	at com.android.launcher3.shortcuts.DeepShortcutTextView.performClick(Unknown Source:53)
E ActivityContext: 	at android.view.View.performClickInternal(View.java:7635)
E ActivityContext: 	at android.view.View.-$$Nest$mperformClickInternal(Unknown Source:0)
E ActivityContext: 	at android.view.View$PerformClick.run(View.java:30167)
E ActivityContext: 	at android.os.Handler.handleCallback(Handler.java:958)
E ActivityContext: 	at android.os.Handler.dispatchMessage(Handler.java:99)
E ActivityContext: 	at android.os.Looper.loopOnce(Looper.java:205)
E ActivityContext: 	at android.os.Looper.loop(Looper.java:294)
E ActivityContext: 	at android.app.ActivityThread.main(ActivityThread.java:8248)
E ActivityContext: 	at java.lang.reflect.Method.invoke(Native Method)
E ActivityContext: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
E ActivityContext: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:971)
```

**Comment by amberin** [comment]
It's the same for me, both with a branch of 1.8.15 and with 1.8.17-beta.1. I wonder how long it's been broken...

**Comment by amberin** [comment]
> Do we know if the following change to the intent config works?
> 
> [14cdfc5#diff-6195cfea4afb55b1d148c4a5bd57bb1dfd92f339dad7ca9da7ceb68e6bdffae3](https://github.com/orgzly-revived/orgzly-android-revived/commit/14cdfc5bed3d43386e009ff33acff502d6e4cd0e#diff-6195cfea4afb55b1d148c4a5bd57bb1dfd92f339dad7ca9da7ceb68e6bdffae3)

Yeah, I wouldn't be suprised if that is it.

**Comment by amberin** [comment]
There is now a 1.8.17-beta.2 version, where I've removed the string variable, and now it works for me. https://github.com/orgzly-revived/orgzly-android-revived/releases/tag/v1.8.17-beta.2

**Comment by amberin** [comment]
Fixed by 247e2c8
